src/components/SearchBar.tsx

```diff
@@ -387,6 +389,11 @@ export function SearchBar({
         screenReaderText={screenReaderText}
         parentQuery={query}
         onToggle={handleToggleDropdown}
+        numItems={
+          (autocompleteResponse?.results.length || 0) +
```

Comment:
We already do this arithmetic above so I was thinking we could use a variable?

Reply:
Oh actually does Dropdown already know the numItems?

Reply:
I don't think so? It has `children` but I'm not sure how to find the size of that lol